docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js

```diff
@@ -11,7 +11,7 @@
                 let offCanvasWidth;
                 const offCanvasCookie = $.cookie('ui_off_canvas_width');
                 if (offCanvasCookie === undefined) {
-                  offCanvasWidth = 500;
+                  offCanvasWidth = auto;
```

Comment:
Comment https://github.com/uiowa/uiowa/pull/4211#issuecomment-929602760 was in regards to this line.